Manual Validation ended with:
> PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Kubecolor.kubecolor__DefaultSource\kubecolor.exe --version [kubecolor] [ERROR] exec: "kubectl": executable file not found in %PATH%

(Automated response - build 931.)

--- a/manifests/k/Kubecolor/kubecolor/0.4.0/Kubecolor.kubecolor.installer.yaml
+++ b/manifests/k/Kubecolor/kubecolor/0.4.0/Kubecolor.kubecolor.installer.yaml
@@ -7,7 +7,10 @@ InstallerType: zip
 NestedInstallerType: portable
 NestedInstallerFiles:
 - RelativeFilePath: kubecolor.exe
-  PortableCommandAlias: kubectl
+  PortableCommandAlias: kubecolor
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Kubernetes.kubectl
 Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/kubecolor/kubecolor/releases/download/v0.4.0/kubecolor_0.4.0_windows_amd64.zip

